### PR TITLE
fix UUID - devel

### PR DESCRIPTION
**Pull request by ObiWahn**

### CODE REVIEW

**Comment by ObiWahn** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/7478/

File: CMakeLists.txt [hunk 1]
```diff
@@ -1140,6 +1140,7 @@ endif ()
 # ZLIB_LIBS
 # ZLIB_INCLUDE_DIR
 
+add_definitions(-DBOOST_UUID_RANDOM_PROVIDER_FORCE_POSIX=1) #fix random provider
```
Comment (dothebart) [inline CMakeLists.txt]: will this do the right thing on windows[tm]?

**Comment by KVS85** [comment]
Tests blue.

File: CMakeLists.txt (hunk 1 above)
Comment (jsteemann) [reply CMakeLists.txt]: Looking at the source (3rdParty/boost/1.71.0/boost/uuid/detail/random_provider_detect_platform.hpp), it shouldn't have any effect on Windows:
``` 
 43 #elif BOOST_OS_WINDOWS
 44 # include <boost/winapi/config.hpp>
 45 # if BOOST_WINAPI_PARTITION_APP_SYSTEM && \
 46      !defined(BOOST_UUID_RANDOM_PROVIDER_FORCE_WINCRYPT) && \
 47      !defined(_WIN32_WCE) && \
 48      (BOOST_USE_WINAPI_VERSION >= BOOST_WINAPI_VERSION_WIN6)
 49 #  define BOOST_UUID_RANDOM_PROVIDER_BCRYPT
 50 #  define BOOST_UUID_RANDOM_PROVIDER_NAME bcrypt
 51 
 52 # elif BOOST_WINAPI_PARTITION_DESKTOP || BOOST_WINAPI_PARTITION_SYSTEM
 53 #  define BOOST_UUID_RANDOM_PROVIDER_WINCRYPT
 54 #  define BOOST_UUID_RANDOM_PROVIDER_NAME wincrypt
 55 # else
 56 #  error Unable to find a suitable windows entropy provider
 57 # endif
 58 
 59 #elif defined(BOOST_UUID_RANDOM_PROVIDER_HAS_GETRANDOM) && !defined(BOOST_UUID_RANDOM_PROVIDER_FORCE_POSIX) && !defined(BOOST_UUID_RANDOM_PROVIDER_DISABLE_GETRANDOM)
 60 # define BOOST_UUID_RANDOM_PROVIDER_GETRANDOM
 61 # define BOOST_UUID_RANDOM_PROVIDER_NAME getrandom
 62 
 63 #elif BOOST_LIB_C_GNU >= BOOST_VERSION_NUMBER(2, 25, 0) && !defined(BOOST_UUID_RANDOM_PROVIDER_FORCE_POSIX) && !defined(BOOST_UUID_RANDOM_PROVIDER_DISABLE_GETENTROPY)
 64 # define BOOST_UUID_RANDOM_PROVIDER_GETENTROPY
 65 # define BOOST_UUID_RANDOM_PROVIDER_NAME getentropy
 66

**Pull request merged**